pkg/controller/controller.go

```diff
@@ -102,8 +105,28 @@ func NewController(clientset kubernetes.Interface, ssclientset ssclientset.Inter
 		},
 	})
 
+	infsecret := informers.NewSharedInformerFactory(clientset, 0).Core().V1().Secrets().Informer()
+	infsecret.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		DeleteFunc: func(obj interface{}) {
+			key, err := cache.MetaNamespaceKeyFunc(obj)
+			if err == nil {
+				ns, name, err := cache.SplitMetaNamespaceKey(key)
+				if err == nil {
+					ssecret, err := ssclientset.BitnamiV1alpha1().SealedSecrets(ns).Get(context.Background(), name, metav1.GetOptions{})
+					if err == nil {
+						key, err := cache.MetaNamespaceKeyFunc(ssecret)
+						if err == nil {
+							queue.Add(key)
+						}
+					}
+				}
+			}
```

Comment:
Nitpick:
Not a fan of nested **ifs** "foreva"
I'd rather fail fast and keep the happy path on the "left lane", straight down.

Reply:
I agree.

Reply:
Modified. Let me know if you think that it is ok for you. Thanks